performance-tests/bench_2/test_controller/main.cpp

```diff
@@ -515,4 +518,12 @@ void handle_reports(const std::vector<Bench::WorkerReport>& parsed_reports, std:
   result_out << std::endl;
 
   consolidated_round_trip_jitter_stats.pretty_print(result_out, "round trip jitter");
+
+  if (total_undermatched_readers ||
+      total_undermatched_writers ||
+      total_out_of_order_data_count ||
+      total_duplicate_data_count) {
+    result = -1;
+  }
```

Comment:
Please don't return a negative number from main: #1362

```suggestion
  result = total_undermatched_readers ||
    total_undermatched_writers ||
    total_out_of_order_data_count ||
    total_duplicate_data_count;
```